docs/qase.md

```diff
@@ -0,0 +1,21 @@
+## [Qase-Reporter](../pkg/qase/README.md)
+## [Qase-Patch-Validation-Create](#qase-patch-validation-create)
+
+
+# Qase-Patch-Validation-Create
+
+### Description
+This job aims to automatically create a patch validation in Qase for a given project and test plan ID.
+
+### All needed variables are already defined or set in run time.
+
+- `QASE_API_TOKEN`: Qase API token to authenticate with the Qase API.
```

Comment:
It will be helpful if we can add a link on how to get this token, is it based on the user?